webapi/src/main/scala/org/knora/webapi/responders/v1/PermissionsResponderV1.scala

```diff
@@ -0,0 +1,853 @@
+    def objectAccessPermissionsForValueGetV1(valueIri: IRI): Future[Option[ObjectAccessPermissionV1]] = {
+        log.debug(s"objectAccessPermissionsForValueGetV1 - resourceIRI: $valueIri")
+        for {
+            sparqlQueryString <- Future(queries.sparql.v1.txt.getObjectAccessPermission(
+                triplestore = settings.triplestoreType,
+                resourceIri = "",
+                valueIri = valueIri
+            ).toString())
+            //_ = log.debug(s"getObjectAccessPermission - query: $sparqlQueryString")
+
+            permissionQueryResponse <- (storeManager ? SparqlSelectRequest(sparqlQueryString)).mapTo[SparqlSelectResponse]
+            //_ = log.debug(s"getDefaultObjectAccessPermissionV1 - result: ${MessageUtil.toSource(permissionQueryResponse)}")
+
+            permissionQueryResponseRows: Seq[VariableResultsRow] = permissionQueryResponse.results.bindings
+
+            groupedPermissionsQueryResponse: Map[String, Seq[String]] = permissionQueryResponseRows.groupBy(_.rowMap("p")).map {
+                case (predicate, rows) => predicate -> rows.map(_.rowMap("o"))
+            }
+            //_ = log.debug(s"getDefaultObjectAccessPermissionV1 - groupedResult: ${MessageUtil.toSource(groupedPermissionsQueryResponse)}")
+
+            permission: Option[ObjectAccessPermissionV1] = if (permissionQueryResponseRows.nonEmpty) {
+
+                val groupedPermissionsQueryResponse: Map[String, Seq[String]] = permissionQueryResponseRows.groupBy(_.rowMap("p")).map {
+                    case (predicate, rows) => predicate -> rows.map(_.rowMap("o"))
+                }
+                val hasPermissions: Set[PermissionV1] = PermissionUtilV1.parsePermissions(groupedPermissionsQueryResponse.get(OntologyConstants.KnoraBase.HasPermissions).map(_.head), PermissionType.OAP)
+                Some(
+                    ObjectAccessPermissionV1(forResource = None, forValue = Some(valueIri), hasPermissions = hasPermissions)
+                )
+            } else {
+                None
+            }
+            _ = log.debug(s"objectAccessPermissionsForValueGetV1 - permission: $permission")
+        } yield permission
+    }
+
+
+    /*************************************************************************/
+    /* DEFAULT OBJECT ACCESS PERMISSIONS                                     */
+    /*************************************************************************/
+
+    /**
+      * Gets all IRI's of all default object access permissions defined inside a project.
+      *
+      * @param projectIRI    the IRI of the project.
+      * @param userProfile the [[UserProfileV1]] of the requesting user.
+      * @return a list of IRIs of [[DefaultObjectAccessPermissionV1]] objects.
+      */
+    private def defaultObjectAccessPermissionsForProjectGetRequestV1(projectIRI: IRI, userProfile: UserProfileV1): Future[DefaultObjectAccessPermissionsForProjectGetResponseV1] = {
+
+        // FIXME: Check user's permission for operation
```

Comment:
Is this fixed? If so, please remove the comment. Otherwise, please make an issue for it on GitHub.